src/app/admin/inventory/(components)/create-category-modal.tsx

```
@@ -0,0 +1,93 @@
+  const { userId } = useUserStore();
+  const form = useForm({
+    mode: 'onBlur',
+    resolver: zodResolver(createNewCategorySchema),
+    defaultValues: {
+      name: ''
+    }
+  });
+
+  const { mutate, isPending } = useMutation({
+    mutationFn: async (formData: CreateNewCategoryType) => createCategory({
+      userId: userId!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)